django/utils/inspect.py

```diff
@@ -64,10 +103,31 @@ def func_accepts_kwargs(func):
     )
 
 
+def func_accepts_var_args(func):
+    if six.PY2:
+        # Not all callables are inspectable with getargspec, so we'll
+        # try a couple different ways but in the end fall back on assuming
+        # it is -- we don't want to prevent registration of valid but weird
```

Comment:
this comment about "prevent registration" is a bit weird since it's related to the registration of signals which the other function is used by, but not this one. I think it would be a good idea to add some unit tests for these new functions so we don't need to debug them through the admindocs tests.